examples/aishell/rnnt/run.sh

```diff
@@ -162,8 +162,18 @@ if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
   # non-streaming model. The default value is -1, which is full chunk
   # for non-streaming inference.
   decoding_chunk_size=
-  ctc_weight=0.5
+  # 1. when beam search these two weight means the prob add weight
+  # 2. when ctc+rnnt nbest ->  rnnt+attn rescoring these two weight means the prob weight rescoring nbest
+  ctc_weight=0.3
+  transducer_weight=0.7
+  # 2. for ctc+rnnt nbest -> rnnt+attn rescoring this weight meas the weight rescoring nbest
+  attn_weight=1.0
+  # 2. for ctc+rnnt nbest this weight means the prob weights decoding nbest
+  search_ctc_weight=0.3
+  search_transducer_weight=0.7
+
```

Comment:
Currently, `ctc_weight` and `search_ctc_weight` are a bit confusing since they are all involved in beam_search. How about this :
```sh
# only used in rescore mode for weighting different scores
rescore_ctc_weight=0.3
rescore_transducer_weight=0.7
rescore_attn_weight=1.0
# only used in beam search, either pure beam search mode OR beam search inside rescoring
beam_search_ctc_weight=0.3
beam_search_transducer_weight=0.7
```

Reply:
ok, I will fix it